.circleci/config.yml

```diff
@@ -1,5 +1,47 @@
 version: 2
 jobs:
+  build-test-linux:
+    docker:
+      - image: matteblair/docker-tangram-linux:latest
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run: make cmake-linux CMAKE_OPTIONS="-DTANGRAM_USE_SYSTEM_FONT_LIBS=1 -DTANGRAM_USE_SYSTEM_GLFW_LIBS=1 -DUNIT_TESTS=1 -DBENCHMARK=1 -GNinja"
+      - run: ninja -C build/linux -j 4
+      - run:
+          name: Run unit tests
+          command: |
+            cd ./build/linux/bin
+            for file in unit/*.out
+              do
+                echo "Running ${file}"
+                $file
+              done
+      - run:
+          name: Run benchmarks
+          command: |
+            # Use a stored tile for testing
+            cp bench/test_tile_10_301_384.mvt build/linux/bin/tile.mvt
+            cd ./build/linux/bin
+            for file in bench/*.out
+              do
+                echo "Running ${file}"
+                $file
+              done
+  build-android:
+    docker:
+      - image: lakoo/android-ndk:25-26.0.1-r15c
```

Comment:
As a note, we should replace this with circle supported android image (sdk + ndk) when this gets available from circle (soonish it seems). Refer: https://github.com/circleci/circleci-images/pull/177